analyzer/src/test/java/com/novoda/sqlite/StringUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package com.novoda.sqlite;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class StringUtilTest {
+    @Test
+    public void shouldCamelizeColumnNames(){
+        assertEquals("ColumnId", StringUtil.camelify("COLUMN_ID"));
+        assertEquals("Columnid", StringUtil.camelify("columnID"));
+        assertEquals("Columnid", StringUtil.camelify("COLUMNID"));
```

Comment:
The test is to document the current state. @devisnik can enlighten us supported naming conventions for column names.